package.json

```diff
@@ -59,7 +59,6 @@
     "ember-fastboot-addon-tests": "0.4.0",
     "ember-load-initializers": "^1.0.0",
     "ember-resolver": "^4.0.0",
-    "ember-source": "~2.13.0",
```

Comment:
Why did you remove ember? One cannot work in development without it!?

Reply:
Sorry, this was an oversight on my part. I think I may have had a cancelled run of `ember-try` that had some side effects. I'll revert this.